.github/workflows/generate-pot.yml

```diff
@@ -15,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - name: WordPress POT Generator
-        uses: varunsridharan/action-wp-pot-generator@^2
+        uses: varunsridharan/action-wp-pot-generator@2
```

Comment:
Nit: The action's readme gives the example of `varunsridharan/action-wp-pot-generator@2.0`.

No big deal, happy to see this merged for a trial run anyway.

Reply:
I'm assuming that this will also work for a 2.1 and I have a PR to the repo. It fixes a bug.